lib/wasm2cretonne-util/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "wasm2cretonne-util"
+version = "0.0.0"
+authors = ["The Cretonne Project Developers"]
+publish = false
```

Comment:
Could you add description and license fields to this and the other Cargo.toml files?